README.md

```diff
@@ -6,7 +6,7 @@ The official implementation of [our paper](https://arxiv.org/abs/2305.08366) "CL
 
 ## Method
 
-CLRerNet features LaneIoU for the target assignment cost and loss functions aiming at the improved quality of confidence scores.   
+CLRerNet features LaneIoU for the target assignment cost and loss functions aiming at the improved quality of confidence scores.
```

Comment:
Could you keep the spaces (for newline) or replace them with `<br>`?

Reply:
Sorry. I didn't check enough. Two trailing single-byte spaces have been restored.
![image](https://github.com/hirotomusiker/CLRerNet/assets/33194443/aa90a1df-470a-4391-a366-02c4c621af49)

Reply:
No problem, thanks!